nextflow_schema.json

```diff
@@ -280,6 +280,24 @@
                 }
             }
         },
+        "generate_samplesheet_options": {
+            "title": "Downstream pipeline samplesheet generation options",
+            "type": "object",
+            "fa_icon": "fas fa-university",
+            "description": "Options for generating input samplesheets for complementary downstream pipelines.",
+            "properties": {
+                "generate_downstream_samplesheets": {
+                    "type": "boolean",
+                    "description": "Turn on generation of samplesheets for downstream pipelines."
+                },
+                "generate_pipeline_samplesheets": {
+                    "type": "string",
+                    "default": "taxprofiler,mag",
+                    "description": "Specify which pipeline to generate a samplesheet for.",
+                    "pattern": "^(taxprofiler|mag)(?:,(taxprofiler|mag)){0,1}"
```

Comment:
OK that's much cleverer than my system :laughing: I always forget about regex :roll_eyes: , will replace in createtaxdb